src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -204,6 +228,7 @@ void reportWavefrontGeneratedData(WavefrontSpan span) {
     // and other invalid metric name characters
     Map<String, String> pointTags = new HashMap<String, String>() {{
       put(OPERATION_NAME_TAG, span.getOperationName());
+      put(COMPONENT_TAG_KEY, span.getComponentTagValue());
```

Comment:
is it useful to have the additional component tag as anyway there is no heartbeat for this?